docs/docs-beta/docs/dagster-plus/access/rbac/audit-logs.md

```diff
@@ -4,4 +4,51 @@ displayed_sidebar: "dagsterPlus"
 sidebar_position: 4
 ---
 
-# Audit logs
\ No newline at end of file
+# Audit logs
+
+The Dagster Plus audit log enables enterprise organizations to track and attribute changes to their Dagster deployment.
+
+This guide walks through how to access the audit log and details the interactions which are tracked in the audit log.
+
+<details>
+<summary>Prerequisites</summary>
+- A Dagster Plus Pro organization
+- An [Organization Admin](/dagster-plus/access/rbac/user-roles-permissions) role in your Dagster Plus organization
+</details>
+
+## View audit logs
+
+Audit logs are available in the Organization settings pane, which can be accessed by clicking on your user icon on the top right side of the screen and selecting Organization settings in the dropdown menu. Once on the Organization settings pane, select Audit log on the top bar.
```

Comment:
Since these are sequential steps, let's put them in a numbered list:

```suggestion
To access the audit logs:

1. Click your user icon at the top right corner of the page.
2. Click **Organization settings**.
3. Click the **Audit log** tab.
```